cvat-ui/src/components/model-runner-modal/detector-runner.tsx

```diff
@@ -118,7 +118,8 @@ function DetectorRunner(props: Props): JSX.Element {
                 <Col span={4}>Model:</Col>
                 <Col span={20}>
                     <Select
-                        placeholder='Select a model'
+                        placeholder={task.dimension ==='2d'?'Select a model' : "No model available"}
```

Comment:
Please, add missed spaces. Doesn't precommit hook with automatic code linting work for you?

Reply:
Also please do not use **--no-verify** flag on commit. The pre-commit hooks must be passed before push

Reply:
We are not using the --no-verify flag on commit, but looks like the pre-commit hook is not giving us any error.

Reply:
Ok, so just put the spaces by hand please:
**==='2d'?'Select a model'** -> **=== '2d' ? 'Select a model'**

Reply:
The space was added as a part of earlier commit, so I do see a space currently in the file, kindly confirm

Reply:
I see. I have approved changes